frame/contracts/src/migration.rs

```diff
@@ -224,13 +224,16 @@ mod v6 {
 			})
 		});
 
+		let nobody = T::AccountId::decode(&mut sp_runtime::traits::TrailingZeroInput::zeroes())
```

Comment:
It's pretty questionable, but that's what the migration did, so we can't alter the logic.

Reply:
It is fine even if `nobody` would be actually `everybody`. For legacy wasm code it is OK if anybody could delete it because it was like this anyways (it was deleted automatically if refcount dropped to `0`).